packages/devextreme/js/ui/multi_view.js

```diff
@@ -213,6 +215,45 @@ const MultiView = CollectionWidget.inherit({
         });
     },
 
+    _getElementAria() {
+        return {
+            role: 'group',
+            'roledescription': messageLocalization.format('dxMultiView-elementAriaRoleDescription'),
+            label: messageLocalization.format('dxMultiView-elementAriaLabel'),
+        };
+    },
+
+    _setElementAria() {
+        const aria = this._getElementAria();
+
+        this.setAria(aria, this.$element());
+    },
+
+    _getItemAria({ itemIndex, itemsCount }) {
+        const aria = {
+            role: 'group',
+            'roledescription': messageLocalization.format('dxMultiView-itemAriaRoleDescription'),
+            label: messageLocalization.format(
+                'dxMultiView-itemAriaLabel',
+                itemIndex + 1,
+                itemsCount,
+            ),
+        };
+
+        return aria;
+    },
+
+    _setItemsAria() {
```

Comment:
[scenarios, perf]
This method is called only on a component's initialization now. Are u sure it's enough for all cases related to a runtime change? I see the base test but not sure if it work in other cases (for example, when new items count is bigger than an old one).
Check plz.
I suggest u move this setting of an aria attribute to the `_updateItemsVisibility` method because:
1. We will be sure that all scenarios are covered
2. It's better for performance

How're u about it?

Reply:
[e88d35eb9fa4d0fcb171faaf710e3d6ef8dba941]